src/test/java/com/quizapp/enterprise/EnterpriseApplicationTests.java

```diff
@@ -64,19 +63,19 @@ void verifyAddAndRemoveGames() {
      */
     @Test
     void verifyCorrectAnswer() {
-        String userGuest = "Blubber";
+        String userGuess = "Blubber";
         String correctAnswer = "Blubber";
 
-        ArrayList<Guess> gec = new GameService().checkGuess(userGuest, correctAnswer);
-        var correct = true;
+        ArrayList<Guess> gec = new GameService().checkGuess(userGuess, correctAnswer);
         for(var guess : gec) {
             if(!guess.IsCorrectLetter) {
-                correct = false;
-                break;
+                fail("User Guess of "
+                        + guess.Letter
+                        + " was not correct");
+                return;
             }
         }
-
-        assertTrue(correct);
+        assertTrue(true);
```

Comment:
Refactored test to use simplified "fail()" assertion when test fails and added message argument to allow the test to provide more valuable feedback in the exception should it fail.
This is recommended by the java ecosystem site referenced below in the "2.4 Testing Condition" section.

source: https://www.baeldung.com/junit-fail